docs/citation.md

```diff
@@ -0,0 +1,18 @@
+# How to Cite
+
+Per [the ScPCA Portal Terms of Use](https://scpca.alexslemonade.org/terms-of-use), users must attribute the data source using any applicable language supplied by data submitters and provided by the Portal when publishing or presenting data or results.
```

Comment:
This deviates a bit more from the wording in the terms of use, but I like the verb `credit` over `attribute` in this case?
```suggestion
Per [the ScPCA Portal Terms of Use](https://scpca.alexslemonade.org/terms-of-use), users must credit the data source using any applicable language supplied by data submitters and provided by the Portal when publishing or presenting data or results.
```

Reply:
I could go either way, so let's see what @dvenprasad thinks!

Reply:
I did not use your exact suggestion here. Instead, I noted that you must cite the individual project(s) and the portal while linking to the relevant sections.